### PR TITLE
Add new API for updating Webhook subscriptions

### PR DESCRIPTION
**Pull request by yadwinder11**
**Asana Task**: https://app.asana.com/0/1199933461955492/1200272129615846/f

## Description
The PR includes the following changes to our API Docs
- Return subscription status as a response of `GET` and `POST` call
- New API `PUT` to update the existing subscription

Screen Shot 2021-05-04 at 3 32 44 PM

### CODE REVIEW

**Review by collinsauve — APPROVED** [review]

File: source/includes/_webhooks.md [hunk 1]
```diff
@@ -26,6 +27,7 @@ Each webhook subscription object has a unique `id`. It also has a `webhook_url`
 | id            | integer  | The unique identifier of the webhook subscription object.                                                                                                                                                             |
 | webhook_url   | string   | The URL to which the webhooks are sent to.                                                                                                                                                                            |
 | subscriptions | string[] | An array of webhook events that are enabled for that endpoint. An empty array indicates subscription to all webhook events. See [below](#supported-webhook-events) for the complete list of supported webhook events. |
+| disabled      | boolean  | If the subscription is disabled, this is true. Otherwise, this is false. |
```
Comment (mimslee32) [inline source/includes/_webhooks.md:30]: `False by default. A subscription may be disabled automatically if ..... or manually.` <- maybe something like that would be more helpful

File: source/includes/_webhooks.md [hunk 2]
```diff
@@ -153,6 +159,45 @@ There is currently a limit of 3 webhook subscriptions.
...
+| ------------- | -------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| webhook_url   | string   | false     | The URL to which the webhooks will be sent to.                                                                                                                                                                                                       |
+| subscriptions | string[] | false    | An array of webhook events that will be enabled for that endpoint. Leave out this parameter or pass an empty array to subscribe to all webhook events. You can find the complete list of supported webhook events [here](#supported-webhook-events). |
+| disabled      | boolean  | false    | Change the status of a subscription. To enable a subscription, provide the value as `true`. Otherwise, provide the value as `false.` |
```
Comment (mimslee32) [inline source/includes/_webhooks.md:195]: Isn't it the opposite? Don't we want `disabled=false` to enable the subscription?

**Review by mimslee32 — CHANGES_REQUESTED** [review]

File: source/includes/_webhooks.md (hunk 2 above)
Comment (yadwinder11) [reply source/includes/_webhooks.md:195]: You are right, Good Catch :)

**Review by mimslee32 — APPROVED** [review]
🚢

**Pull request merged**